### PR TITLE
Added picocrypt json

### PR DESCRIPTION
**Pull request by Duckle29**
Picocrypt is a tiny tool for encrypting files

### CODE REVIEW

File: bucket/picocrypt.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.17",
+    "description": "A very small, very simple, yet very secure encryption tool",
+    "homepage": "https://evansu.cc/picocrypt",
```
Comment (Ash258) [inline bucket/picocrypt.json:4]: Absolutely useless homepage....

image

File: bucket/picocrypt.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.17",
+    "description": "A very small, very simple, yet very secure encryption tool",
```
Comment (Ash258) [inline bucket/picocrypt.json:3]: Does not tell end user enything. What does it encrypt?? What is it for?

File: bucket/picocrypt.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "homepage": "https://evansu.cc/picocrypt",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/HACKERALERT/Picocrypt/releases/download/1.17/Picocrypt.exe",
+    "hash": "ae5becaa3b2e6398cc13287bb23afab95684975b55ebcd4946dd92f6e4974450",
+    "bin": "Picocrypt.exe",
```
Comment (Ash258) [inline bucket/picocrypt.json:8]: What are command line options?

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by Duckle29** [comment]
I am no longer using this tool

**Pull request closed without merging**